### PR TITLE
Add fix to registry for field selector

### PR DESCRIPTION
**Pull request by asinner**

### CODE REVIEW

File: src/data/FieldRegistry.tsx [hunk 1]
```diff
@@ -27,4 +27,14 @@ export class FieldRegistry {
 
         this[field.type] = field;
     }
+
+    public getFields(): IFieldDef[] {
+        const fields: IFieldDef[] = []
+        for (var key in this) {
```
Comment (niuyunsi) [inline src/data/FieldRegistry.tsx]: Should we use 'let' instead of 'var'

File: src/components/FieldSelector.tsx [hunk 2]
```diff
@@ -16,12 +16,10 @@ export class FieldSelector extends React.PureComponent<IFieldSelectorProps, IFie
 
     render() {
         let options = [];
-        for (let key in this.props.registry) {
-            const field = this.props.registry[key].field;
-            const displayName = this.props.registry[key].displayName;
-            options.push(<FieldSelectorOption field={field} key={key} label={displayName} />)
-        }
-
+        this.props.registry.getFields().forEach(def => {
```
Comment (linweigao) [inline src/components/FieldSelector.tsx]: Should we use map then?

Comment (asinner) [reply src/components/FieldSelector.tsx]: Done

File: src/data/FieldRegistry.tsx (hunk 1 above)
Comment (asinner) [reply src/data/FieldRegistry.tsx]: done

**Pull request merged**